<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delete_query</class>
 <widget class="QDialog" name="delete_query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>160</width>
     <height>139</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="delete_s1">
      <property name="text">
       <string>Delete Account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_s2">
      <property name="text">
       <string>Delete Review</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_s3">
      <property name="text">
       <string>Cancel Order</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_s4">
      <property name="text">
       <string>Delete Item</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
